src/machine/aot/aot.x64.c

```diff
@@ -1224,7 +1317,8 @@ int aot_memory_write(AotContext* context, AotValue address, AotValue v, uint32_t
   if (ret != DASM_S_OK) { return ret; }
 
   | mov rdx, size
-  | call ->check_write
+  | mov rcx, 1
```

Comment:
A common trick here, is that we can write `mov ecx, 1` for the same result, but it will be encoded in a shorter instruction.

Reply:
Fixed.